Composer/packages/client/src/utils/os.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Check this file out:
Composer/Composer/packages/client/src/recoilModel/utils/fontUtil.ts

Reply:
Heh. Didn't see that. I'll deprecate that and use the new function.